src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import styles from './Dropdown.module.css';
+
+export default function Dropdown({ text }) {
+  const [val, setVal] = useState('');
+  console.log(val);
+  const status = ['Not Submitted', 'Submitted', 'No Callback', 'Interview Scheduled', 'Interviewing', 'Offer Received'];
```

Comment:
Can you move this outside the component? It's specific to one use case of the dropdown

Reply:
move what outside of dropdown? can you be more specific?